todo_app/data/trello_items.py

```diff
@@ -0,0 +1,84 @@
+from todo_app.data.item import Item
+
+import requests
+import os
+
+__boardId = os.getenv('TRELLO_BOARDID')
+__key = os.getenv('TRELLO_KEY')
+__token = os.getenv('TRELLO_TOKEN')
+__default_list = 'To Do'
+
+def change_status(title, status):
+
+    cards = __get_board_cards(__boardId)
+    cardId = None
+    for card in cards:
+        if card['name'] == title:
+            cardId = card['id']
+    if cardId == None:
+        return
+
+    lists = __get_board_lists(__boardId)
+    listId = None
+    for list in lists:
+        if list['name'] == status:
+            listId = list['id']
+    if listId == None:
+        return
+    
+    __set_card_list(cardId, listId)
+
+
+def get_statuses():
+    lists = __get_board_lists(__boardId)
+    for list in lists:
+        yield list['name']
+
+
+def get_items() -> list[Item]:
+    lists = __get_board_lists(__boardId)
+    for card in __get_board_cards(__boardId):
+        listName = __get_list_name(lists, card['idList'])
+        yield Item(card['id'], listName, card['name'])
+  
+
+def get_item(id) -> Item:
+    return None
+
+
+def add_item(title):
+    lists = __get_board_lists(__boardId)
+    listId = None
+    for list in lists:
+        if list['name'] == __default_list:
+            listId = list['id']   
+    __add_card(listId, title)
+
+
+
+def save_item(item) -> Item:
+    return item
+
+
+def __add_card(listId, name):
+    response = requests.post(f"https://api.trello.com/1/cards?key={__key}&token={__token}&name={name}&idList={listId}")
```

Comment:
It's generally better to let the library build the query params, for the same reason you wouldn't concatenate strings into an SQL query, e.g. what if name is `Some_title&desc=Injected_description`? `requests.post` has a `params` keyword argument for this.

Reply:
done